projects/fitness_recommendations/requirements-app.txt

```diff
@@ -0,0 +1,12 @@
+flytekit==0.30.0
+flytekitplugins-pandera==0.30.0
+kaggle==1.5.12
+flyteidl
+geopy
+joblib
+pandas
+pandera
+requests
+sklearn
+timezonefinder
+dataclasses
```

Comment:
i plan to use the file @Thejaswi05 created, but wanted to have my own environment for what I was doing first. This file will not be included in my branch